crawl-ref/source/aptitudes.h

```diff
@@ -276,6 +276,45 @@ static const species_skill_aptitude species_skill_aptitudes[] =
     APT(SP_KOBOLD,          SK_INVOCATIONS,     1),
     APT(SP_KOBOLD,          SK_EVOCATIONS,      2),
 
+    // SP_MOUNTAIN_DWARF
+    APT(SP_MOUNTAIN_DWARF,  SK_FIGHTING,        2),
+    APT(SP_MOUNTAIN_DWARF,  SK_SHORT_BLADES,   -2),
+    APT(SP_MOUNTAIN_DWARF,  SK_LONG_BLADES,    -2),
+    APT(SP_MOUNTAIN_DWARF,  SK_AXES,            2),
+    APT(SP_MOUNTAIN_DWARF,  SK_MACES_FLAILS,    4),
+    APT(SP_MOUNTAIN_DWARF,  SK_POLEARMS,       -1),
```

Comment:
+4 ?!?! Do we really want another merfolk...? These apts seem slightly extreme, but maybe that is ok.

Reply:
The difference is, they're small, so they can't use great maces.

Reply:
I didn't realise they were small. I would point out that axes are bad for small races, especially when compared to maces: broad axes are two-handed for 13 damage, morningstars are one-handed for 13 damage and eveningstars are one-handed for 15 damage.

Reply:
That's a good point, so I changed morningstars/eveningstars to be 2h for small races. :-)